### PR TITLE
Catch known test warning

### PR DESCRIPTION
**Pull request by duytnguyendtn**
### Description

There was an unforeseen clash between #1583 and #1564, where a known astropy units warning introduced in #1564 was popping up in #1583's test. I copied over the warnings ignore, and also matched them so that the warnings ignore is more specific

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [ ] Is a change log needed? If yes, is it added to `CHANGES.rst`?
- [ ] Is a milestone set?
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: jdaviz/configs/specviz/plugins/line_analysis/tests/test_line_analysis.py [hunk 1]
```diff
@@ -43,6 +43,7 @@ def test_plugin(specviz_helper, spectrum1d):
             assert len(result_dict.get('uncertainty')) > 0
 
 
+@pytest.mark.filterwarnings('ignore', match="'W/m2/m' contains multiple slashes")
```
Comment (pllim) [inline jdaviz/configs/specviz/plugins/line_analysis/tests/test_line_analysis.py:46]: I think you want something like this. See https://docs.pytest.org/en/7.1.x/how-to/capture-warnings.html#pytest-mark-filterwarnings

```suggestion
@pytest.mark.filterwarnings(r'ignore:W/m2/m' contains multiple slashes')
```

Comment (duytnguyendtn) [reply jdaviz/configs/specviz/plugins/line_analysis/tests/test_line_analysis.py:46]: I thought I've seen both syntaxes used? Happy to switch

Comment (pllim) [reply jdaviz/configs/specviz/plugins/line_analysis/tests/test_line_analysis.py]: Some characters are special in regex. You can play with https://regex101.com/ or maybe just generalize it to something like this:

```python
r"ignore:.* contains multiple slashes"
```

Comment (duytnguyendtn) [reply jdaviz/configs/specviz/plugins/line_analysis/tests/test_line_analysis.py]: Worked! I also forgot to catch the redshift slider warning. Let's see if it passes below (it passes on my local copy)

Comment (pllim) [reply jdaviz/configs/specviz/plugins/line_analysis/tests/test_line_analysis.py]: Weird that this wasn't caught in the PR CI.

**Review by rosteen — APPROVED** [review]
Tests passed, thanks!

File: jdaviz/configs/specviz/plugins/line_analysis/tests/test_lineflux.py [hunk 2]
```diff
@@ -92,7 +93,7 @@ def test_cubeviz_collapse_fluxunits(spectrum1d_cube_custom_fluxunit, spectra_flu
             assert u.Unit(lineflux_result['unit']) == expected_lineflux_results[spectra_fluxunit]
 
 
-@pytest.mark.filterwarnings('ignore')
+@pytest.mark.filterwarnings(r"ignore:.* contains multiple slashes")
```
Comment (pllim) [inline jdaviz/configs/specviz/plugins/line_analysis/tests/test_lineflux.py:96]: Belated 👍 . It is nice to narrow down the ignored warnings. Thanks!

**Pull request merged**